### DISCUSSION TITLE
Mounting an NFS share from within the Home Assistant VM

### DISCUSSION THREAD

**Discussion by darrendavid** [tteck/Proxmox#2630] [Help Needed]
I used the script to spin up a Home Assistant VM within Proxmox and it worked like a charm. I'm having issues mounting NFS shares from within the VM, though - I get a "Permission Denied" error from the command line. 

`[core-ssh /]$ mount -t nfs 10.0.1.10:/mnt/data/shared /mnt
mount: mounting 10.0.1.10:/mnt/data/shared on /mnt failed: Permission denied`

These same NFS shares are able to be mounted from an Ubuntu LXC created from these same scripts (after installing `nfs-common`), and I've removed any and all network filtering from the share so it should be good to go. Is there anything about the way this VM is created that needs to be tweaked to allow mounting of NFS shares?

**Comment by darrendavid** [comment]
Bump

**Comment by TychoWerner** [comment]
HA has built in mounting of NFS, check https://my.home-assistant.io/redirect/storage/
